src/inspector/view/qml/MuseScore/Inspector/general/appearance/AppearanceSettings.qml

```diff
@@ -70,6 +70,7 @@ Column {
 
         offset: model ? model.offset : null
         isSnappedToGrid: model ? model.isSnappedToGrid : null
+        isVerticalOffsetAvailable: model ? model.isVerticalOffsetAvailable : null
```

Comment:
It would be nicer to use `false` instead of `null` here, since this is about booleans rather than objects:
```suggestion
        isVerticalOffsetAvailable: model ? model.isVerticalOffsetAvailable : false
```
(functionally it won't make any difference because JavaScript doesn't care very much about types, but semantically it is cleaner)